pkgs/applications/display-managers/sddm/sddm-catppuccin.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, lib, fetchFromGitHub, flavor ? "mocha" }:
+stdenv.mkDerivation {
+  pname = "sddm-catppuccin-theme";
+  version = "0.0.0b0";
+
+  dontBuild = true;
```

Comment:
Is this actually required? Is stdenv detecting some makefile?

Reply:
this wasn't required. It must've been an artifact from another theme.